docs/architecture/c-sharp-client-library.md

```diff
@@ -67,20 +67,80 @@ The user can expect to either have a primitive type (such as an int) or one of t
 
 ### Handlers
 
-There will be two handlers for each of the data types returned:
+There are currently two handlers for each of the data types returned:
```

Comment:
Should the location handler that merged in the library be added to these docs as well?

Reply:
Added information about LocationHandler. Mentioned it separately than the 2 data source handlers we have.